plugins/maven/src/main/java/org/wildfly/swarm/plugin/maven/PackageMojo.java

```diff
@@ -200,8 +203,12 @@ public void error(String msg, Throwable t) {
                 .forEach(tool::additionalModule);
 
         try {
-            File jar = tool.build(finalName + (this.hollow ? "-hollow" : ""), Paths.get(this.projectBuildDir));
-
+            String jarFinalName = finalName + (this.hollow ? "-hollow" : "") + "-swarm";
```

Comment:
What's produced if `finalName` isn't set?

Reply:
final String finalName = this.project.getBuild().getFinalName();
so the default finalName of maven. If that's not set, it will default to ${project.artifactId}-${project.version} afaik.